description: use wine64 as the wine binary name on arm64
author: Michael Gilbert <mgilbert@debian.org>
forwarded: not-needed

--- a/configure.ac
+++ b/configure.ac
@@ -2029,6 +2029,7 @@ dnl **** Platform-specific checks ****
 case "$HOST_ARCH,$PE_ARCHS" in
   x86_64,*i386*) wine_binary="wine" ;;
   x86_64,*) wine_binary="wine64" ;;
+  aarch64,*) wine_binary="wine64" ;;
   *) wine_binary="wine" ;;
 esac
 AC_SUBST(WINELOADER_PROGRAMS,"$wine_binary")
